cli/main.rs

```diff
@@ -273,6 +274,58 @@ async fn compile_command(
   Ok(0)
 }
 
+async fn init_command(
+  _flags: Flags,
+  init_flags: InitFlags,
+) -> Result<i32, AnyError> {
+  let dir = if let Some(dir) = &init_flags.dir {
+    let dir = std::env::current_dir()?.join(dir);
+    std::fs::create_dir_all(&dir)?;
+    dir
+  } else {
+    std::env::current_dir()?
+  };
+
+  let mod_ts = r#"export function add(a: number, b: number): number {
+  return a + b;
+}
+
+if (import.meta.main) {
+  console.log("Add 2 + 3", add(2, 3));
+}
```

Comment:
Maybe we should exclude the if statement, but keep the console.log? I think the if statement might confuse some beginners.

Reply:
I think it would be also confusing if you run the test and then you get a console.log :/

Reply:
Wouldn’t that be obvious looking at the code?

Reply:
And the way the test runner works, it would show as test output, letting users know they get this sort of stuff managed for them.

Reply:
> And the way the test runner works, it would show as test output, letting users know they get this sort of stuff managed for them.

Not really - since it would be a console.log at the top level of the file it would be printed just after "Check ..." like so:

```
Check file:///Users/ib/dev/deno/mod_test.ts
Add 2 + 3 5
running 1 test from ./mod_test.ts
addTest ... ok (4ms)

ok | 1 passed | 0 failed (24ms)
```

I'm fine with removing `import.meta.main` conditional, or we could add a comment that explains its purpose or linking to the manual.